share/functions/cd.fish

```diff
@@ -3,10 +3,13 @@
 #
 function cd --description "Change directory"
     set -l MAX_DIR_HIST 25
+    set -l options h/help P
```

Comment:
We also want a long-version like our `pwd`s `--physical`.

So this should be `P/physical`.

Reply:
Ok and what about -L for the long version?